src/System.Private.Xml/src/System/Xml/Core/HtmlEncodedRawTextWriter.cs

```diff
@@ -70,7 +61,7 @@ public override void WriteDocType(string name, string pubid, string sysid, strin
 
             RawText("<!DOCTYPE ");
 
-            // Bug: Always output "html" or "HTML" in doc-type, even if "name" is something else
+            // Bug 114337: Always output "html" or "HTML" in doc-type, even if "name" is something else
```

Comment:
the id doesn't exist even in our internal bug db, remove :-) (please try to make it so that the generated files has as little diff as possible)